src/app/components/header/header.component.ts

```diff
@@ -8,21 +10,26 @@ import { Component, OnInit } from '@angular/core';
 
 
 export class HeaderComponent implements OnInit {
+      public isAuth: boolean;
       public buttons = [
-        {name:'Home', active:false, path:'home'},
-        {name:'Register', active:false, path: 'signup'},
-        {name:'Login', active:false, path:'login'},
-        {name:'Account Summary', active:false, path:'secure-home'},
-        {name:'Profile', active:false, path:'profile'},
-        
+        {name:'Home', active:false, path:'home', secure:false},
+        {name:'Register', active:false, path: 'signup', secure:false},
+        {name:'Login', active:false, path:'login', secure: false}       
       ];
 
-  constructor() { 
-    
+  constructor(protected loginService: LoginService) { 
   }
 
   ngOnInit() {
     
+  }
+  authenticated() {
```

Comment:
@jmilleriec this looks like it is common across all components? we can yank it out in the common component.